pyramid_ogcapi/json2html.py

```diff
@@ -0,0 +1,281 @@
+    -------
+    convert(json_input)
+        Converts JSON to HTML.
+    """
+
+    def __init__(
+        self,
+        attributes_json_output: Optional[list[str]] = None,
+        table_attributes: Optional[dict[str, str]] = None,
+    ):
+        """
+        Initialize the converter.
+        """
+        self.attributes_json_output = attributes_json_output or []
+        if table_attributes is not None and not isinstance(table_attributes, dict):
+            raise TypeError("Table attributes must be either a `dict` or `None`.")
+
+        self._table_opening_tag = "<table{:s}>".format(
+            JsonConverter._dict_to_html_attributes(table_attributes)
+        )
+
+    def convert(self, json_input: Json) -> str:
+        """
+        Convert JSON to HTML Table format.
+
+        Parameters
+        ----------
+        json_input : dict
+            JSON object to convert into HTML.
+
+        Returns
+        -------
+        str
+            String of converted HTML.
+        """
+
+        if len(json_input) == 1:
+            key = list(json_input.keys())[0]
+            if key in self.attributes_json_output:
+                return f"<b>{html.escape(key)}</b>: {html.escape(json.dumps(json_input[key]))}"
+            elif not isinstance(json_input[key], list) and not isinstance(json_input[key], dict):
+                return f"<b>{html.escape(key)}</b>: {html.escape(json_input[key])}"
+
+        html_output = self._table_opening_tag
+        for key, value in iter(json_input.items()):
+            html_output += f"<tr><th>{self._markup(key):s}</th>"
+            if key in self.attributes_json_output:
+                html_output += f"<td>{html.escape(json.dumps(value))}</td>"
+            else:
+                if isinstance(value, list):
+                    html_output += self._maybe_club(value)
+                else:
+                    html_output += self._markup_table_cell(value)
+            html_output += "</tr>"
+        html_output += "</table>"
+        return html_output
+
+    def _markup_table_cell(self, value: Json) -> str:
+        """
+        Wrap the generated HTML in table cell `<td></td>` tags.
+
+        Parameters
+        ----------
+        value : object
+            Object to place in the table cell.
+
+        Returns
+        -------
+        str
+            String of HTML wrapped in table cell tags.
+        """
+        return f"<td>{self._markup(value):s}</td>"
+
+    def _markup_header_row(self, headers: list[str]) -> str:
+        """
+        Create a row of table header items.
+
+        Parameters
+        ----------
+        headers : list
+            List of column headers. Each will be wrapped in `<th></th>` tags.
+
+        Returns
+        -------
+        str
+            Table row of headers.
+        """
+        return "<tr><th>" + "</th><th>".join([html.escape(h) for h in headers]) + "</th></tr>"
+
+    @staticmethod
+    def _dict_to_html_attributes(d: Json) -> str:
+        r"""
```

Comment:
```suggestion
        """
```

Reply:
Without that, I get the error:
[D301(pydocstyle), None] Use r""" if any backslashes in a docstring